util/metrics/src/service_metrics.rs

```diff
@@ -48,19 +49,19 @@ use std::str;
 /// e.g., calc_service.duration_sum{method="add"} = 6
 #[derive(Clone)]
 pub struct ServiceMetrics {
-    /// Number of requests made by methods
+    /// Number of requests made by each gRPC method tracked
     num_req: IntCounterVec,
 
-    /// Number of error responses for methods
+    /// Number of error responses for each gRPC method tracked
     num_error: IntCounterVec,
 
-    /// Number of GRPC status codes for methods
+    /// Count of gRPC status codes for each gRPC method tracked
```

Comment:
nit: why did this get changed to `Count`? Seems inconsistent with the variable name of `num` and the other member docs using `Number`.

as an aside I generally lean toward "count" over "number". A "number" can mean; how many (i.e. a counter), or the number 3 (an id).  While "count" seems to be less ambiguous, unless you're dealing with count chocula.

Reply:
Changed to count it creates a Prometheus Counter

Reply:
The other comments that are changed here should be changed to "Count" as well, then ;-)

Reply:
Can do